source/optional_features/digital_wallets/index.html.md.erb

```diff
@@ -0,0 +1,69 @@
+---
+title: Digital wallets
+weight: 136
+---
+
+# Digital wallets
+
+You can enable [Google Pay](https://pay.google.com/intl/en_uk/about/) and [Apple Pay](https://www.apple.com/uk/apple-pay/) [external links] to take payments from your users.
+
+This is an invitation-only feature currently in private beta. [Contact the GOV.UK Pay team](https://docs.payments.service.gov.uk/support_contact_and_more_information/#contact-us) to request this feature for your service. After GOV.UK Pay has approved your request to participate in this private beta, GOV.UK Pay will email you a link to enable this feature. 
+
+You can only enable Google Pay and Apple Pay if you have a [live Worldpay account](/switching_to_live/set_up_a_live_worldpay_account/#set-up-a-live-worldpay-account).
+
+## Enable Apple Pay
+
+Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/).
+
+1. Click on the link in the email from GOV.UK Pay 
+
+1. Select __Enable__ next to Apple Pay.
+
+1. On the __Enable Apple Pay__ screen, select __Yes, turn on Apple Pay__.
+
+## Enable Google Pay
+
+To enable Google Pay, you must enter a unique Google Pay merchant ID into your GOV.UK Pay account.
+
+You get your Google Pay merchant ID from your Worldpay merchant admin interface.
+
+1. If required, speak to your Worldpay account manager to upgrade your Worldpay merchant admin interface, so that the interface has Google Pay functionality.
+
+1. Sign in to your [Worldpay merchant admin interface](https://secure.worldpay.com/sso/public/auth/login.html) [external link].
+
+1. Go to __Integration__ and select the __Pay with Google__ tab.
+
+1. Under __Generate unique merchant ID__, select __Generate__.
+
+1. Copy the generated merchant ID.
+
+Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/).
+
+1. Click on the link in the email from GOV.UK Pay 
+
+1. Select __Enable__ next to Google Pay.
+
+1. On the __Enable Google Pay__ screen, select __Yes, turn on Google Pay__.
+
+1. Enter the generated merchant ID and select __Submit__.
+
+## Secure authentication
+
+A user can securely authenticate digital wallet transactions that require secure authentication by either:
+
+* enabling secure authentication in their digital wallet
+* securely authenticating those transactions separately using [3D Secure authentication](https://www.pcisecuritystandards.org/documents/FAQs_for_PCI_3DS_Core_Security_Standard.pdf) [external link]
+
+Apple Pay enforces secure authentication, so users should not have to complete 3D Secure authentication during Apple Pay transactions.
+
+Google Pay does not enforce secure authentication. 
```

Comment:
Technically this is correct but I’m not too sure about it. We’re introducing this concept of “secure authentication” that’s distinct from 3D Secure authentication but I’m not certain all readers will fully grasp the specificity of “secure authentication” in phrases like “Google Pay does not enforce secure authentication.” (The distinction is actually a bit blurry: when secure authentication is used with a digital wallet, it gets processed using the 3D Secure mechanism even though the user never sees 3D Secure.)

I think the important part for services is whether or not users will see 3D Secure, so I’d anchor the text around that. Something like this (phrasing obviously needs improvement):

“Users paying with Apple Pay never need to complete 3D Secure authentication because Apple Pay requires transactions to be authenticated using a secure mechanism like a fingerprint.

“Google Pay supports authenticating transactions using a fingerprint or similar secure mechanism, but does not require it. Users who do not use Google Pay’s secure authentication may have to complete 3D Secure authentication.”

I think the 3D Secure link is probably too technical for most of our target audience. I mentioned fingerprint authentication as an example because that’s probably something readers are familiar with and can relate to.